cypress/support/utils.js

```diff
@@ -1,3 +1,6 @@
+import dedent from 'dedent';
```

Comment:
```suggestion
/// <reference types= "cypress" />
import dedent from 'dedent';
```

<img width="592" alt="types" src="https://user-images.githubusercontent.com/3297808/223889043-07b47c0e-7f00-4b12-a2d6-c04770ecad16.png">

Reply:
```
// eslint-disable-next-line spaced-comment
/// <reference types="cypress" />
```
🙄